docs/aws/README.md

```diff
@@ -67,12 +96,14 @@ eksctl create iamidentitymapping \
   --group system:nodes
 ```
 
-### Install Karpenter
+Now, Karpenter can send requests for new EC2 instances to AWS. 
```

Comment:
Now, Karpenter can send requests for new EC2 instances to AWS and those instances can connect to your cluster. 

These two pieces are core to our permission model.

Reply:
👍